packages/styles/scss/components/masthead/_masthead.scss

```diff
@@ -314,7 +314,7 @@ $search-transition-timing: 95ms;
 
       &:focus,
       &:active {
-        &[aria-selected='true'] {
+        &[selected='true'] {
```

Comment:
@andy-blum thanks! Looking at the Percy snapshots, the React Masthead L0 and L1 also uses the `aria-selected` attribute. Could we adjust the React version as well?

<img width="1153" alt="Screen Shot 2021-09-01 at 9 02 39 AM" src="https://user-images.githubusercontent.com/54281166/131676073-46620112-6a92-4b51-8fed-d0eed800fb4b.png">

Reply:
or it may make sense to switch both React and Web components to use a classname instead for styling